package.json

```diff
@@ -16,6 +16,7 @@
         "@types/chai": "^4.2.11",
         "@types/express": "^4.17.6",
         "@types/fs-extra": "^9.0.1",
+        "@types/stats": "^0.16.30",
```

Comment:
I'm missing stats.js itself in the dependencies

Reply:
Done